### PR TITLE
Try to change doctrine/orm version for supporting 2.8 (PHP 8 support).

### PR DESCRIPTION
**Pull request by GothShoot**
Try to use doctrine/orm 2.8 with job queue, this is the last step to php 8 support.

### CODE REVIEW

**Comment by makasim** [comment]
Have you tested it ?

**Comment by GothShoot** [comment]
Need to setup a work environment because it needs env var to work with your testing config.
I give you a response when I have tested 👍

**Comment by makasim** [comment]
You should not require this repo directly. fork read only package, apply your changes and require it in composer. 

Once done, backport changes to this repo.

**Comment by GothShoot** [comment]
My fork run on my app (just installing and clear the cache)

**Comment by makasim** [comment]
Good, thanks

File: pkg/job-queue/composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "enqueue/enqueue": "^0.10",
         "enqueue/null": "^0.10",
         "queue-interop/queue-interop": "^0.8",
-        "doctrine/orm": "~2.7.4",
+        "doctrine/orm": "~2.7.4 | ^2.8",
```
Comment (deguif) [inline pkg/job-queue/composer.json:13]: Can not this be simplified to `"^2.7.4"` only?

**Comment by makasim** [comment]
fixes https://github.com/php-enqueue/enqueue-dev/issues/1151

**Pull request merged**